keps/sig-network/20181024-service-topology.md

```diff
@@ -175,6 +176,13 @@ When a Node is updated, PodLocator controller will first check if its labels are
 
 When a Node is deleted, PodLocator controller will reset the NodeName and NodeLabels of all the PodLocators whose corresponding Pods running on it.
 
+### EndpointSlice controller changes
+
+The EndpointSlice controller will be updated to set values corresponding to the
+TopologyKeys field on Services. When `kubernetes.io/hostname` is specified as a
```

Comment:
Why make this different?  Nodes generally have the hostname label.  It's not special.

Reply:
Good catch, this was based on the initial implementation that got what it could from the pod before pulling labels from the node, but that does not make sense. Fixed now, thanks!